Changeing name conventions

TobiasABay opened the pull request:
# Description

*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*

Fixes #\<issue_no>

## Type of change
*Delete unchecked boxes (only for Type of change)*

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration*


**Development Configuration**
*Type "flutter --version" and "dart --version" in your CMD to check versions.*

* Flutter version:
* Dart version:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas, if necessary
- [ ] I have made corresponding changes to the documentation, if necessary
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, if necessary
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have Acceptance Tested this on an iOS device
- [ ] I have Acceptance Tested this on an Android device

TobiasABay committed:
Changeing name conventions

Dra3e commented on foodplanner/lib/components/meal_box_empty.dart:
```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class Mealboxempty extends StatelessWidget {
```
Skulle mene at MealBoxEmpty ville være convention

Dra3e commented on foodplanner/lib/components/meal_box_content.dart:
```diff
@@ -3,17 +3,14 @@ import 'package:foodplanner/components/image.dart';
 import 'package:foodplanner/pages/landing_page_children_se_madpakke.dart'; // Update with the correct import
-import 'package:foodplanner/components/dateTimePicker.dart';
 
 class Mealboxcontent extends StatelessWidget {
```
Skulle mene at MealBoxContent ville være convention

TobiasABay replied:
Du har ret - Det er blevet omdøbt :)

Dra3e reviewed (changes requested):
Der er lige klassenavn i meal_box_content og meal_box_empty, der skal rettes til at følge convention

TobiasABay committed:
renamed:    foodplanner/lib/pages/profilePage.dart -> foodplanner/lib/pages/profile_page.dart
	renamed:    foodplanner/lib/pages/settingsPage.dart -> foodplanner/lib/pages/settings_page.dart

TobiasABay committed:
modified:   foodplanner/lib/components/meal_box.dart
	modified:   foodplanner/lib/components/meal_box_content.dart

Dra3e commented on foodplanner/test/service_tests/ingredient_test.mocks.dart:
```diff
@@ -48,8 +48,8 @@ class _FakeStreamedResponse_1 extends _i1.SmartFake
 /// See the documentation for Mockito's code generation for more information.
-class MyMockClient extends _i1.Mock implements _i2.Client {
-  MyMockClient() {
+class MockClient extends _i1.Mock implements _i2.Client {
```
Den skal stadig hede MyMockClient

Dra3e commented on foodplanner/test/service_tests/meal_test.mocks.dart:
```diff
@@ -48,8 +48,8 @@ class _FakeStreamedResponse_1 extends _i1.SmartFake
 ///
 /// See the documentation for Mockito's code generation for more information.
-class MyMockClient extends _i1.Mock implements _i2.Client {
-  MyMockClient() {
```
Den skal stadig hedde MyMockClient

Dra3e commented on foodplanner/test/service_tests/packed_ingredient_test.mocks.dart:
```diff
@@ -48,8 +48,8 @@ class _FakeStreamedResponse_1 extends _i1.SmartFake
 /// See the documentation for Mockito's code generation for more information.
-class MyMockClient extends _i1.Mock implements _i2.Client {
-  MyMockClient() {
+class MockClient extends _i1.Mock implements _i2.Client {
```
Den skal stadig hede MyMockClient

TobiasABay committed:
modified:   foodplanner/test/service_tests/ingredient_test.mocks.dart
	modified:   foodplanner/test/service_tests/meal_test.mocks.dart

TobiasABay committed:
modified:   foodplanner/test/service_tests/packed_ingredient_test.mocks.dart

TobiasABay committed:
modified:   foodplanner/lib/components/meal_box.dart
	modified:   foodplanner/lib/components/meal_box_empty.dart
